test/e2e/cruise/network/network_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2019-2024 The Liqo Authors
```

Comment:
Add an afterSuite to delete namespace if test fail. Also check for namespace deletion